### PR TITLE
Dictionary updates ― English-Japanese-Chinese (2023.12.12)

### PR DESCRIPTION
**Pull request by SleepyAsh0191**
Long time no see. 

I saw your tweet earlier, so I decided to contribute some words related to 4.2. 
Organizing words is really tiring, and I deeply understand the state you were in when you were doing it before. 
This words organization took a long time, and I also involved a friend of mine in the research.

There might be some missing words or misclassifications in this code contribution, please understand.

The content from the 4.3 live broadcast will be organized later.

### CODE REVIEW

**Comment by SleepyAsh0191** [comment]
still some grammar error, checking

**Comment by SleepyAsh0191** [comment]
@xicri I'm now ready!

**Review by xicri — APPROVED** [review]
Thanks so much for adding words including Japanese!
Looks good, I'll merge this.

File: dataset/dictionary/enemies.json5 [hunk 1]
```diff
@@ -2179,4 +2179,24 @@
...
+    zhCN: "维护机关·澄金领队型",
+    pronunciationJa: "メンテマシナリーおうごんとうそつがた",
+    tags: [ "fontaine", "enemy" ],
+  },
+  {
+    en: "All-Devouring Narwhal",
+    ja: "呑星の鯨",
+    zhCN: "吞星之鲸",
+    pronunciationJa: "どんせいのくじら",
+    tags: [ "fontaine", "enemy" ],
```
Comment (xicri) [inline dataset/dictionary/enemies.json5:2200]: @SleepyAsh0191 Sorry for the review comment after merge.
Isn't this a weekly boss? (Actually, I haven't played the 4.2 Archon Quest yet, so I might be wrong.)
If All-Devouring Narwhal is a weekly boss, `enemy-boss` is the better categorization than `enemy`.

Comment (SleepyAsh0191) [reply dataset/dictionary/enemies.json5:2200]: sorry mischeck it's enemy-boss

Comment (SleepyAsh0191) [reply dataset/dictionary/enemies.json5:2200]: > @SleepyAsh0191 Sorry for the review comment after merge. Isn't this a weekly boss? (Actually, I haven't played the 4.2 Archon Quest yet, so I might be wrong.) If All-Devouring Narwhal is a weekly boss, `enemy-boss` is the better categorization than `enemy`.

i will fix it in next contribute

Comment (xicri) [reply dataset/dictionary/enemies.json5:2200]: No problem, thanks! 😊

**Pull request merged**